pulpcore/cli/core/worker.py

```diff
@@ -0,0 +1,27 @@
+import click
+
+from pulpcore.cli.common.context import (
+    PulpContext,
+    pass_pulp_context,
+)
+from pulpcore.cli.common.generic import list_command, show_command, href_option, name_option
+from pulpcore.cli.core.context import PulpWorkerContext
+
+
+@click.group()
+@pass_pulp_context
+@click.pass_context
+def worker(ctx: click.Context, pulp_ctx: PulpContext) -> None:
+    ctx.obj = PulpWorkerContext(pulp_ctx)
+
+
+filter_options = [
+    click.option("--name"),
+    click.option("--name-contains", "name__contains"),
+    click.option("--name-icontains", "name__icontains"),
+    click.option("--missing/--not-missing", default=None),
+    click.option("--online/--not-online", default=None),
+]
+
+worker.add_command(list_command(decorators=filter_options))
+worker.add_command(show_command(decorators=[href_option, name_option]))
```

Comment:
Whenever i think, this is a lot of boilerplate, i need to remind myself, that it would be ca. 8-16 times the code if spelled out here in more detail...